media-video/pipewire/metadata.xml

```diff
@@ -10,11 +10,20 @@
 		<name>Andreas Sturmlechner</name>
 	</maintainer>
 	<upstream>
-		<remote-id type="github">PipeWire/pipewire</remote-id>
+		<bugs-to>https://gitlab.freedesktop.org/pipewire/pipewire/-/issues</bugs-to>
+		<changelog>https://gitlab.freedesktop.org/pipewire/pipewire/-/releases</changelog>
```

Comment:
Must this be a literal file or would a probably auto-generated URL with fancy HTML also count? It's updated on release but I have not verified it has no subtly changing bits.

Reply:
Apparently nothing is automatically parsing it, so it shouldn't be a problem.